<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Verification</class>
 <widget class="QWidget" name="Verification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆验证</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/new/prefix1/Resources/人脸识别.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 149, 255);
</string>
       </property>
       <property name="text">
        <string>人脸识别系统</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 149, 255);</string>
       </property>
       <property name="text">
        <string>请输入密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="bt_ok">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style:outset;
border-radius: 8px;
background-color: rgb(0, 133, 221);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/prefix1/Resources/校验_check.svg</normaloff>:/new/prefix1/Resources/校验_check.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_close">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style:outset;
border-radius: 8px;
background-color: rgb(0, 133, 221);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/prefix1/Resources/关闭_close.svg</normaloff>:/new/prefix1/Resources/关闭_close.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
